### PR TITLE
Add real TPS to the loadbot output

### PR DESCRIPTION
**Pull request by ZeljkoBenovic**
# Description

This PR adds the real number of transactions per second to the loadbot output, by dividing total number of transactions by total time in seconds it took to process all transactions.

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [x] I have tested this code
- [ ] I have updated the README and other relevant documents (guides...)
- [ ] I have added sufficient documentation both in code, as well as in the READMEs

### CODE REVIEW

**Comment by mrwillis** [comment]
Just chiming in here @ZeljkoBenovic and more of a discussion. I think one should consider normalized TPS by considering the gas usage. Maybe another metric. Naturally pure ETH transfers would produce the highest TPS, but I think the loadbot has some transaction modes which take higher gas, and would produce lower TPS for the same gas limit.

**Review by brkomir — APPROVED** [review]

**Comment by ZeljkoBenovic** [comment]
I totally agree with @mrwillis . This PR introduces output which shows the estimated TPS in regards to regular ETH transfers only. It doesn't take into consideration gas and stuff like that. This will just show estimated tps by dividing total number of transactions with total number of seconds it took to complete the run. We should definitely introduce a more precise estimation of TPS in the future.

File: command/loadbot/loadbot_command.go [hunk 1]
```diff
@@ -372,9 +374,14 @@ type LoadbotResult struct {
 	TurnAroundData    TxnTurnAroundData    `json:"turnAroundData"`
 	BlockData         TxnBlockData         `json:"blockData"`
 	DetailedErrorData TxnDetailedErrorData `json:"detailedErrorData,omitempty"`
+	RealTPS           uint64               `json:"realTps"`
 }
 
 func (lr *LoadbotResult) extractExecutionData(metrics *Metrics) {
+	// calculate real transactions per second value
+	// by deviding total transactions by total time in seconds
+	//nolint:lll
```
Comment (zivkovicmilos) [inline command/loadbot/loadbot_command.go]: Please remove the nolint clause and break up the next line into several lines for readability, you can do it in Go

File: command/loadbot/loadbot_command.go [hunk 2]
```diff
@@ -433,6 +440,11 @@ func (lr *LoadbotResult) Output() string {
 		fmt.Sprintf("Transactions failed|%d", lr.CountData.Failed),
 	}))
 
+	buffer.WriteString("\n\n[REAL TPS]\n")
```
Comment (zivkovicmilos) [inline command/loadbot/loadbot_command.go]: Please rename this section to something like `APPROXIMATE TPS`

**Review by zivkovicmilos — APPROVED** [review]
Looks good, please see the conversations 💯

File: command/loadbot/loadbot_command.go [hunk 3]
```diff
@@ -433,6 +440,11 @@ func (lr *LoadbotResult) Output() string {
...
+	buffer.WriteString(helper.FormatKV([]string{
+		fmt.Sprintf("Real number of transactions per second|%d", lr.RealTPS),
```
Comment (zivkovicmilos) [inline command/loadbot/loadbot_command.go]: Please add this TPS value to the json output as well 👍

**Review by Kourin1996 — APPROVED** [review]
Looks good to me 👌

**Review by lazartravica — APPROVED** [review]

File: command/loadbot/loadbot_command.go (hunk 2 above)
Comment (ZeljkoBenovic) [reply command/loadbot/loadbot_command.go]: Renamed f9886aed1bef4a43315d3b30f970cda6e1c9e8c2

File: command/loadbot/loadbot_command.go (hunk 3 above)
Comment (ZeljkoBenovic) [reply command/loadbot/loadbot_command.go]: Added json output f9886aed1bef4a43315d3b30f970cda6e1c9e8c2

File: command/loadbot/loadbot_command.go (hunk 1 above)
Comment (ZeljkoBenovic) [reply command/loadbot/loadbot_command.go]: Removed linter clause and broke that line of code to multiple lines f9886aed1bef4a43315d3b30f970cda6e1c9e8c2

**Pull request merged**